Gds/docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Gds Command and Control Tool
+
+## 1. Introduction
+
+This is a tool intended to be used with FPrime dpeloyments to exchange commands, telemetry, events and other messages.
+This tool includes logging capability and can be connected to several adapters to communicate with ground station equipment.
+
+## 2. Message Field Configuration
+
+Certain deployments may alter the fields of messsages for various reasons including memory conservation.
+To make the corresponding changes to the Gds for successful communication, 
+please edit the Gds/src/fprime_gds/common/utils/config_manager.py file accordingly. 
+Each field in the _set_defaults() function corresponds to a specific compile time flag for an FPrime deployment.
+Bellow is a table of corresponding fields that must match between deployment and Gds in order for successful communication to occur.
+
+Gds Field | FPrime Flag
+----------- | ----------- |
+self.__prop['types']['msg_len'] | TOKEN_TYPE
+self.__prop['types']['msg_desc'] | FwPacketDescriptorType
+self.__prop['types']['op_code'] | FwOpcodeType
+self.__prop['types']['ch_id'] | FwChanIdType
+self.__prop['types']['event_id'] | FwEventIdType
+self.__prop['types']['pkt_id'] | Unkown
+self.__prop['types']['key_val'] | unknown
+
+It is important to note that the default value of all of these fprime fields is U32, which is the behavior when none of these compile time flags are set in any CMakeLists.txt fields.
```

Comment:
Ideally, you'd describe this as "setting values in FpConfig.hpp" as for now that would be how we'd set these.

Reply:
I changed the documentation to focus on FpConfig.hpp edits instead of CMakeLists flag defines for changing the fields